### PR TITLE
feat: found declarative annotations from go-module packages

### PR DESCRIPTION
**Pull request by easonlin404**
**Describe the PR**
Will also found declarative annotations from go-module packages(outside $GOPATH).

**Relation issue**
#267

### CODE REVIEW

**Comment by easonlin404** [comment]
@pei0804 please help review it.

**Comment by pei0804** [comment]
:eyes:

**Comment by pei0804** [comment]
I looks like good job.

File: parser.go [hunk 1]
```diff
@@ -97,18 +100,48 @@ func SetMarkdownFileDirectory(directoryPath string) func(*Parser) {
...
+	var stdout, stderr strings.Builder
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("execute go list command, %s", err)
+	}
+
+	outStr, _ := stdout.String(), stderr.String()
+	if outStr[0] == '_' {
```
Comment (pei0804) [inline parser.go]: What is the situation like this?

Comment (easonlin404) [reply parser.go]: go list command will show like `_/{GOPATH}/src/{YOUR_PACKAGE}` when NOT enable GO MODULE.

File: parser.go [hunk 2]
```diff
@@ -1237,12 +1271,31 @@ func (parser *Parser) getAllGoFileInfo(searchDir string) error {
...
+	return nil
+}
+
 func (parser *Parser) visit(path string, f os.FileInfo, err error) error {
 	if err := parser.Skip(path, f); err != nil {
 		return err
 	}
 
 	if ext := filepath.Ext(path); ext == ".go" {
+		// Printf("visit path:%s\n", path)
```
Comment (pei0804) [inline parser.go]: It forget delete?

File: parser.go (hunk 1 above)
Comment (pei0804) [reply parser.go]: I want to write comment.
What do you think?

**Comment by pei0804** [comment]
I think good flow.

File: parser.go (hunk 1 above)
Comment (easonlin404) [reply parser.go]: yep, I will add comment for it.

**Comment by pei0804** [comment]
LGTM

**Comment by Nerzal** [comment]
Nice one :)

**Pull request merged**